### ISSUE TITLE
Support for X3: Farnham's Legacy

### ISSUE THREAD

**Issue by AldebaranPrime** [nick-crowley/X-Studio#1]
Is it possible to publish an update of X-Studio-IDE to support the new  X3: Farnham's Legacy MSCI-Syntax and the /addon2/-folder?

**Comment by MegaBurn** [comment]
Yes, shouldn't be difficult.  I posted some notes with a very rough 
overview in the FL S&M Questions thread 
(https://forum.egosoft.com/viewtopic.php?p=5048064#p5048064), a starting 
point, not comprehensive.  I would try to do it but I don't have Visual 
Studio or a VC build environment setup.

For FL syntax, the Python script needs overhauled to extract data from 
FL XML data (I wrote that on 5/2 from the command list posted on the 
wiki, sorry I haven't updated it, been busy, will make some time over 
the next few days).  I haven't seen anything in game data which 
correlates parameter names or ID's to commands, generic "value" could be 
used but this would cause problems.  The X-Studio 2 update posted by 
Deniskos appears to contain an XML file with updated command syntax, 
should be fairly easy to convert that to XS1 format, he previously 
posted syntax with "value" for params, I'd assume he updated this 
manually by reading off the in-game editor.

Would also be a good idea to move the command syntax data to an external 
file for easier future updates.
